salon_creator/lib/models/salon.dart

```diff
@@ -0,0 +1,33 @@
+class Salon {
+  String salonName;
```

Comment:
```suggestion
  String name;
```
Since we're already in the **Salon** class, it is understood that **name** is **Salon.name**